tests/aws/aws_kubernetes.rs

```diff
@@ -165,17 +58,17 @@ fn create_eks_cluster_in_us_east_2() {
 
     let cloudflare = dns_provider_cloudflare(&context);
 
-    let mut file = File::open("tests/assets/eks-options.json").unwrap();
+    let mut file = File::open("cloned-engine/tests/assets/eks-options.json").unwrap();
```

Comment:
wait why? there is no cloned-engine in this repo

Reply:
oups